module/README.md

```diff
@@ -50,6 +50,8 @@ for users that prefer a file based approach.
 | Property                             | Environment Variable          | Description                                              |
 |--------------------------------------|-------------------------------|----------------------------------------------------------|
 | `zipkin.collector.otel.http.enabled` | `COLLECTOR_HTTP_OTEL_ENABLED` | `false` disables the HTTP collector. Defaults to `true`. |
+| `zipkin.collector.otel.http.resource-attribute-prefix` | `COLLECTOR_OTEL_RESOURCE_ATTRIBUTE_PREFIX` | The prefix to use when converting otel resource attributes to span annotations. The default is to not prefix anything. |
+| `zipkin.collector.otel.http.log-event-name-attribute` | `COLLECTOR_OTEL_LOG_EVENT_NAME_ATTRIBUTE` | The otel attribute name that indicates whether to convert the upcoming Log Event to a Span. By default, `event.name` is used. |
```

Comment:
I don't think it will ever be anything but event.name. Do we need a config around this? It might be more helpful to use config to decide what to do when it is `event.name`, like record the annotation with value == event.name attribute (search is usable) vs value == jsonification of the message (high cardinality so more cost, not usable in search)

Reply:
Ah, I seem to have misunderstood your comment.
The stability of the `event.name` attribute is "experimental", so I thought that might change in the future.
It's not a very useful configuration, so I'll remove it.

Reply:
Most things in otel conventions seem to be experimental ;)

Reply:
Drop the configuration in https://github.com/openzipkin-contrib/zipkin-otel/pull/26/commits/004492a24567fc5af4dd6ee85884cfbe7a726d15